### ISSUE TITLE
[Feature] Search for Java in the "java" folder on Windows

### ISSUE THREAD

**Issue by RuinTD** [PolyMC/PolyMC#358]
On Linux, PolyMC has a feature where it searches for Java installs in a "java" folder in PolyMC's folder:
https://github.com/PolyMC/PolyMC/blob/87cf38a3770dd7f55fbae79ca3a4958155289af7/launcher/java/JavaUtils.cpp#L420-L421

It would be nice if this feature could be added to Windows.

**Comment by DioEgizio** [comment]
Yeah this is a relic from MultiMC, on Windows it just searches on the registries.
This should be changed, and we need it for #7
